### ISSUE TITLE
How to bypass the registration form on joining the meeting

### ISSUE THREAD

**Issue by JAGANPS** [praharshbhatt/flutter_zoom#1]
Hi @praharshbhatt  I have been following your Code for zoom plugin but the issues are that I am not able to run iOS application properly because it throws the error like this 
missing required architecture x86_64 in file
    /Users/Desktop/flutter/.pub-cache/git/flutter_zoom_plugin-88d1fa2685f28d67410d46e0944
    61a0a030d3192/ios/MobileRTC.framework/MobileRTC (2 slices)
    Undefined symbols for architecture x86_64:
      "_OBJC_CLASS_$_MobileRTCSDKInitContext", referenced from:
          objc-class-ref in SwiftFlutterZoomPlugin.o
      "_OBJC_CLASS_$_MobileRTC", referenced from:
          objc-class-ref in SwiftFlutterZoomPlugin.o
      "_kMeetingParam_MeetingPassword", referenced from:
          flutter_zoom_plugin.ZoomView.joinMeeting(call: __C.FlutterMethodCall, result: (Any?) ->
          ()) -> () in SwiftFlutterZoomPlugin.o
      "_kMeetingParam_Username", referenced from:
          flutter_zoom_plugin.ZoomView.joinMeeting(call: __C.FlutterMethodCall, result: (Any?) ->
          ()) -> () in SwiftFlutterZoomPlugin.o
      "_OBJC_CLASS_$_MobileRTCMeetingStartParam4WithoutLoginUser", referenced from:
          objc-class-ref in SwiftFlutterZoomPlugin.o
      "_kMeetingParam_MeetingNumber", referenced from:
          flutter_zoom_plugin.ZoomView.joinMeeting(call: __C.FlutterMethodCall, result: (Any?) ->
          ()) -> () in SwiftFlutterZoomPlugin.o
    ld: symbol(s) not found for architecture x86_64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but
    the range of supported deployment target versions is 9.0 to 14.3.99. (in target
    'flutter_zoom_plugin' from project 'Pods')
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but
    the range of supported deployment target versions is 9.0 to 14.3.99. (in target 'Flutter'
    from project 'Pods')

Could not build the application for the simulator.
Error launching application on iPhone

And in Android how to by-pass the registration forum while joining the meeting.

**Comment by praharshbhatt** [comment]
Are you running on a simulator?
If so, what branch are you using?

**Comment by JAGANPS** [comment]
Yes I am using simulator  the branch I am using for android is 0.0.8 and for iOS I use master
I am getting this error on iOS which I am not able to create the build itself

the error I get for iOS is 



Launching lib/main.dart on iPhone 6 Plus in debug mode...
 
Running pod install...                                              2.4s
Running Xcode build...                                                  
                                                   
Xcode build done.                                           15.5s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    ld: warning: ignoring file
    /Users/jaganps/Desktop/flutter/.pub-cache/git/flutter_zoom_plugin-08999fdfc4
    ce035781d3078d445d6cc97ccb3f6f/ios/MobileRTC.framework/MobileRTC, missing
    required architecture x86_64 in file
    /Users/jaganps/Desktop/flutter/.pub-cache/git/flutter_zoom_plugin-08999fdfc4
    ce035781d3078d445d6cc97ccb3f6f/ios/MobileRTC.framework/MobileRTC (2 slices)
    Undefined symbols for architecture x86_64:
      "_OBJC_CLASS_$_MobileRTC", referenced from:
          objc-class-ref in SwiftFlutterZoomPlugin.o
      "_kMeetingParam_Username", referenced from:
          flutter_zoom_plugin.ZoomView.joinMeeting(call: __C.FlutterMethodCall,
          result: (Any?) -> ()) -> () in SwiftFlutterZoomPlugin.o
      "_kMeetingParam_MeetingNumber", referenced from:
          flutter_zoom_plugin.ZoomView.joinMeeting(call: __C.FlutterMethodCall,
          result: (Any?) -> ()) -> () in SwiftFlutterZoomPlugin.o
      "_OBJC_CLASS_$_MobileRTCSDKInitContext", referenced from:
          objc-class-ref in SwiftFlutterZoomPlugin.o
      "_kMeetingParam_MeetingPassword", referenced from:
          flutter_zoom_plugin.ZoomView.joinMeeting(call: __C.FlutterMethodCall,
          result: (Any?) -> ()) -> () in SwiftFlutterZoomPlugin.o
    ld: symbol(s) not found for architecture x86_64
    clang: error: linker command failed with exit code 1 (use -v to see
    invocation)
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is
    set to 8.0, but the range of supported deployment target versions is 9.0 to
    14.3.99. (in target 'flutter_zoom_plugin' from project 'Pods')
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is
    set to 8.0, but the range of supported deployment target versions is 9.0 to
    14.3.99. (in target 'Runner' from project 'Runner')
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is
    set to 8.0, but the range of supported deployment target versions is 9.0 to
    14.3.99. (in target 'Flutter' from project 'Pods')

Could not build the application for the simulator.
Error launching application on iPhone 6 Plus.

**Comment by Kabariya** [comment]
is it fixed or any workaround for this error?

**Comment by JAGANPS** [comment]
Yeah wrote my own plugin they are working fine now :)

**Comment by Kabariya** [comment]
can you tell me how can i fix it?

**Comment by JAGANPS** [comment]
Android or iOS
